apiary.apib

```diff
@@ -798,8 +808,8 @@ form of choices.
                     }
                 ]
             }
-            
-## Answer [/customers/{cid}/surveys/{sid}/questions/{qid}/answer]
+
+## Answer [/customers/{cid}/answers]
```

Comment:
I moved the business logic here: surveys got questions that are answered by customers. Legit owner of answer is customer.

Of course it would require from you to get a full list of questions from the webservice, full list of answers from the customer and intersect them to produce a view out of them.